packages/ui/src/molecules/RadioGroup/RadioOption.tsx

```diff
@@ -0,0 +1,49 @@
+import type { InputHTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+import { Radio } from '../..'
+import { useRadioOption } from './useRadioOption'
+
+export interface RadioOptionProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, 'type'> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Attribute to set aria-label option as required.
+   */
+  label: string
+  /**
+   * Value to be emitted when radio is clicked.
+   */
+  value: string | number
+}
+
+const RadioOption = forwardRef<HTMLInputElement, RadioOptionProps>(
+  function RadioOption(
+    { label, value, children, testId = 'store-radio-option', ...otherOptions },
+    ref
+  ) {
+    const { name, selectedValue, onChange } = useRadioOption()
+
+    return (
+      <label aria-label={label} data-store-radio-option>
+        <Radio
+          data-store-radio-option-item
```

Comment:
Just to double-check that we have 2 `data-attributes` to identify the component since the Radio component has its own `data-store-radio` as well. If this is the desired behavior, just ignore my comment here hahaha

Reply:
Sim, esse é o comportamento, Ícaro explicou [nesse comentário](https://github.com/vtex/faststore/pull/1033#discussion_r756214596)

Reply:
data-radio-option-item following the [docs](https://github.com/vtex/faststore/blob/master/docs/components-architecture.md)